### DISCUSSION TITLE
endpoints.tsのimportが.jsになっているのは何故？

### DISCUSSION THREAD

**Discussion by himantotopuhu** [misskey-dev/misskey#11229] [Q&A] [answered]
https://github.com/misskey-dev/misskey/blob/develop/packages/backend/src/server/api/endpoints.ts
↑で大量にimportされているファイルは.jsとつけられていますが、実際に存在するのは全てtsファイルです。
.jsがつけられている理由は何かあるのですか？

**Comment by acid-chicken** [comment, marked as answer]
See https://www.typescriptlang.org/docs/handbook/esm-node.html
